extensions/binding/org.eclipse.smarthome.binding.homematic/NOTICE.txt

```diff
@@ -0,0 +1,19 @@
+This content is produced and maintained by the Eclipse SmartHome project.
```

Comment:
The filename should be "NOTICE", not "NOTICE.txt", see e.g. https://github.com/eclipse/smarthome/blob/master/extensions/binding/org.eclipse.smarthome.binding.astro/NOTICE